lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/services/LibertyWorkspace.java

```diff
@@ -107,7 +109,7 @@ public void initConfigFileList() {
     public void scanXMLforInclude(Path filePath) {
         try {
             String content = new String(Files.readAllBytes(filePath));
-            Matcher m = Pattern.compile("<include.+location=[\"\'](.+)[\"\']/>").matcher(content);
+            Matcher m = Pattern.compile("<include[^<]+location=[\"\'](.+)[\"\']/>").matcher(content);
```

Comment:
Can you add a comment that explains this regex and what it will match exactly? Explain what is in group 1 since you use that below.

Reply:
Group 1 is limited to the contents of what `location` gets defined to.

`[^<]` matches anything not a `<`, including white spaces/newlines and therefore more thorough than `.`. The idea being that it would consume everything up to `location` and as long as that attribute was defined in the same `include` element.

I just find out attributes can be defined as child elements though, so I'll make an adjustment shortly.
I'll add comments to the code as well.